examples/all_the_things.py

```diff
@@ -89,25 +90,28 @@ def dimensions(test):
 
 
 def attachments(test):
-  test.Attach('test_attachment', 'This is test attachment data.')
-  test.AttachFromFile('example_attachment.txt')
+  test.attach('test_attachment', 'This is test attachment data.')
```

Comment:
I'm down to write a changelog, but I'd like you guys to vet what I did with the examples in the [restructuring PR (on my fork)](https://github.com/jettisonjoe/openhtf/pull/2) so I can make sure what I'm writing up is what we end up using.